README.md

```diff
@@ -46,234 +46,59 @@ Git Proxy deploys custom push protections and policies on top of Git. It is a hi
 Git Proxy is built with a developer-first mindset. By presenting simple-to-follow remediation instructions in the CLI/Terminal, it minimises the friction of use and adoption, and keeps developers focused on what matters; committing and pushing code.
 
 
-## Installation
+## Getting Started 🚀
 
-To install Git Proxy, you must first install [Node.js](https://nodejs.org/en/download). Then, use the [npm](https://www.npmjs.com/) package manager:
+Install & run git-proxy (requires [Nodejs](https://nodejs.org/en/download/)):
 
 ```bash
-npm install -g @finos/git-proxy
+$ npx -- @finos/git-proxy
+Need to install the following packages:
+  @finos/git-proxy@1.1.0
+Ok to proceed? (y) 
+...
+creating user
+        user=admin,
+        gitAccount=none
+        email=admin@place.com,
+        admin=true
+Service Listening on 8080
+Listening on 8000
 ```
 
-To install a specific version of Git Proxy, append the version to the end of the `install` command:
+Clone a repository, set the remote to the Git Proxy URL and push your changes:
 
 ```bash
-npm install -g @finos/git-proxy@1.1.0
+$ git clone https://github.com/octocat/Hello-World.git && cd Hello-World
+$ gh repo fork
```

Comment:
@coopernetes - okay assuming that folks will know what this is?

Reply:
Trying my best here to give a shortest possible introduction to git-proxy. Most of the minutiae is captured in the doc site's quick start guide so didn't want to reinvent here and would rather direct users to that guide which is more end-to-end and comprehensive. Alternatively, we can have just `Install & run git-proxy` section above and omit the push example. Something like:

````markdown
## Getting Started
Install & run git-proxy:

```bash
$ npx -- @finos/git-proxy
Need to install the following packages:
  @finos/git-proxy@1.1.0
Ok to proceed? (y) 
...
creating user
        user=admin,
        gitAccount=none
        email=admin@place.com,
        admin=true
Service Listening on 8080
Listening on 8000
```
You can now push commits directly through the proxy server and start implementing your own organization's controls. For a more comprehensive demo, please see the [Quick start guide](https://git-proxy.finos.org/docs/category/quickstart) on the project's documentation.
````

Reply:
@coopernetes - I like the length and simplicity of your introduction, we should keep this. I'm more referring to the use of the `gh` CLI in the steps?

Reply:
Good point. I will leave a small bash-style comment.